samples/java/src/main/java/br/com/example/leadIntegration/models/Lead.java

```diff
@@ -0,0 +1,23 @@
+package br.com.example.leadIntegration.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
@giovannicaldeira maybe you can change all this annotation for this https://projectlombok.org/features/Data

Reply:
@robsonoduarte Change annotation to Data 

```java
@Data
@Builder
public class Lead {
...
}
```